api/nakadi-event-bus-api.yaml

```diff
@@ -995,11 +1014,42 @@ definitions:
         type: string
         default: 'random'
 
+      compatibility_mode:
```

Comment:
will Nakadi check that compatibility mode is respected when schema is updated? Or is it something that should be guaranteed by event owner?